core/src/main/java/org/elasticsearch/index/query/IndexQueryParserService.java

```diff
@@ -77,7 +77,7 @@ protected QueryShardContext initialValue() {
 
     final IndexCache indexCache;
 
-    final IndexFieldDataService fieldDataService;
+    protected IndexFieldDataService fieldDataService;
```

Comment:
I don't think this is needed now due to @s1monw 's recent PR ? https://github.com/elastic/elasticsearch/pull/13381